src/webui/webapplication.cpp

```diff
@@ -774,6 +777,33 @@ void WebApplication::action_command_bottomPrio()
     BitTorrent::Session::instance()->bottomTorrentsPriority(hashes);
 }
 
+void WebApplication::action_command_setLocation()
+{
+    CHECK_URI(0);
+    CHECK_PARAMETERS("hashes" << "location");
+
+    QStringList hashes = request().posts["hashes"].split("|");
```

Comment:
>  Why give up better ways just to be in line with older code (that nobody has interest updating it)?

I have/had interest in updating it. When you point out changes like these I am happy to tackle the entire file at once. However nitpicking every line of a PR makes me a lot less eager to do so.